src/OnboardingSPA/components/LivePreview/BlockPreview/index.js

```diff
@@ -53,7 +62,16 @@ const BlockPreview = ( {
 	}, [ storedPreviewSettings ] );
 
 	return (
-		<div className={ `live-preview__container-${ styling }` }>
+		<div className={`live-preview__container-${styling }  live-preview-container` }>
```

Comment:
Is there a reason we have an extra `live-preview-container` here?

Reply:
So I added that in case someone uses the BlockPreview component someday without the browser style outside.

Reply:
Removed the common Global CSS. Thanks